qiskit/assembler/disassemble.py

```diff
@@ -152,6 +178,18 @@ def _experiments_to_circuits(qobj):
             if conditional and name != "bfunc":
                 _inst.c_if(conditional["register"], conditional["value"])
                 conditional = {}
+        pulse_lib = qobj.config.pulse_library if hasattr(qobj.config, "pulse_library") else []
+        parametric_pulses = (
+            qobj.config.parametric_pulses if hasattr(qobj.config, "parametric_pulses") else []
+        )
+        if hasattr(qobj.config, "calibrations"):
+            circuit.calibrations = dict(
+                **circuit.calibrations, **_qobj_to_circuit_cals(qobj, pulse_lib, parametric_pulses)
+            )
```

Comment:
There's a dict method that does this:
```suggestion
            circuit.calibrations.update(_qobj_to_circuit_cals(qobj, pulse_lib, parametric_pulses)
            )
```
This will work for lines 190-192 as well

Reply:
yea that was the initial implementation, but for some reason circuit.calibrations.update() never seemed to mutate circuit.calibrations. oddly enough, it would do just fine some some dummy dict `foo` here, which makes me think it has something to do with being an attr of a class instance. that's why we ultimately tried other options that gave the expected behavior and settled on the existing one. am i missing something obvious or is this not how dict.update should behave here? is there another approach that might work better than the existing one with this in mind?

![Screen Shot 2021-05-19 at 7 24 53 PM](https://user-images.githubusercontent.com/25019973/118897253-e4bcf400-b8d7-11eb-8e22-701f69d09a5c.png)
![Screen Shot 2021-05-19 at 7 25 17 PM](https://user-images.githubusercontent.com/25019973/118897292-f3a3a680-b8d7-11eb-9d40-a5b969a971ed.png)